src/index.css

```diff
@@ -149,29 +162,30 @@ hr {
   line-height: normal;
   letter-spacing: -0.18px;
   color: #888;
+  font-family: 'Montserrat', sans-serif;
 }
 
 .bTitle {
   background-color: #fff;
   border-radius: 4px;
-  width: 50%;
+  width: 45%;
   padding: 15px;
   margin-left: 20px;
   border: 1px solid #e8e8e8;
 }
 
 .bAuthor {
-  width: 27%;
-  margin: 0 0 0 3%;
-  height: 48px;
-  padding: 0 0 0 1%;
-  border: 1px solid #e8e8e8;
+  background-color: #fff;
   border-radius: 4px;
+  width: 35%;
+  padding: 15px;
+  margin-left: 20px;
+  border: 1px solid #e8e8e8;
 }
```

Comment:
- [ ] Kindly make sure to display the progress bar as shown in the below video attached as per the requirements. 
_The original design is shown on the left side and your on the right side._

https://user-images.githubusercontent.com/70269507/172825281-6dacc8bd-7da9-4883-b7bd-cddad7ef7c3c.mov